_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual-steps.md

```diff
@@ -0,0 +1,137 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## 1. Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through an IAM role called `GruntworkAccessRole` in each of the accounts through the `AdministratorAccess` policy. Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
+`__current__` (i.e., `--account "__current__"`).
+
+
+**Important**: Never use the `AdministratorAccess` AWS managed policy with any users, groups, or roles. It gives full access to all resources. Instead, to give access to administrators, use a policy that allows full IAM privileges (e.g. `iam:*`) on all resources. Our Landing Zone solution created a `iam-admin` group in the Security account. [More information about the existing groups in the `iam-groups` module.](https://github.com/gruntwork-io/terraform-aws-security/tree/main/modules/iam-groups)
```

Comment:
This paragraph remains pretty confusing for me: 
1. It's not clear to what extent the recommendations here are specific to the CIS-Benchmark, or are just general Gruntwork AWS best-practices. Is it Gruntwork saying to never use `AdministratorAccess`? Or does the CIS-Benchmark require that you have no policies attached which grant `AdministratorAccess` / `*:*`?
2. It's not clear what we're asking the user to do about the `iam-admin` group in the security account
3. It feels like the last sentence of the paragraph doesn't belong

Reply:
These are good questions! I re-worded the text trying to answer them.